tests/workflow_test.go

```diff
@@ -127,6 +129,288 @@ func (s *WorkflowTestSuite) TestStartWorkflowExecution() {
 	})
 }
 
+func (s *WorkflowTestSuite) TestStartWorkflowExecution_UseExisting() {
+	id := "functional-start-workflow-use-existing-test"
```

Comment:
nit: `testcore.RandomizeString(s.T().Name())` and don't bother with the naming, it's nice for duplicating tests later.

Same for other variables here.

Reply:
Or just use testvars.